lib/vehicle.rb

```diff
@@ -6,13 +6,19 @@ class Vehicle
               :make,
               :model,
               :engine
+              
+  attr_accessor :registration_date,
+                :plate_type
+              
 
   def initialize(vehicle_details)
     @vin = vehicle_details[:vin]
     @year = vehicle_details[:year]
     @make = vehicle_details[:make]
     @model = vehicle_details[:model]
     @engine = vehicle_details[:engine]
+    @registeration_date 
+    @plate_type
```

Comment:
It seems like this is working as is, but I would expect these attributes to be defined with an assignment operator, even if the values start as nil or an empty string.
```ruby
@registration_date = nil
@plate_type = nil
```